site/app/templates/submission/regrade/Discussion.twig

```diff
@@ -1,6 +1,46 @@
+<div><h3>Regrade Request
+{% if btn_type == "request" %}
+    </h3></div>
```

Comment:
Instead of opening outside the if and closing inside the if, just move the whole thing into the if. That way the tags open and close in the same block and it's much easier to confirm every opened tag is closed.